src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -143,27 +146,31 @@ class WorkspaceSettingsPage extends React.Component {
                                     onImageRemoved={this.removeAvatar}
                                 />
                             </OfflineWithFeedback>
-                            <TextInput
-                                label={this.props.translate('workspace.editor.nameInputLabel')}
-                                containerStyles={[styles.mt4]}
-                                onChangeText={name => this.setState({name})}
-                                value={this.state.name}
-                                hasError={!this.state.name.trim().length}
-                                errorText={this.state.name.trim().length ? '' : this.props.translate('workspace.editor.nameIsRequiredError')}
-                            />
-
-                            <View style={[styles.mt4]}>
-                                <Picker
-                                    label={this.props.translate('workspace.editor.currencyInputLabel')}
-                                    onInputChange={currency => this.setState({currency})}
-                                    items={this.getCurrencyItems()}
-                                    value={this.state.currency}
-                                    isDisabled={hasVBA}
+                            <OfflineWithFeedback
+                                pendingAction={lodashGet(this.props.policy, 'pendingFields.generalSettings')}
+                                onClose={() => {}}
```

Comment:
Rather than passing a no-op, just refactor the component to not require this prop, and then use a default prop of a no-op.

Reply:
Good call.